Ginger/Ginger/UserControlsLib/PieChart/StatusColorSelector.cs

```diff
@@ -71,7 +71,7 @@ public Brush SelectBrush(object item, int index)
                 case "Started":
                 case "Running":
                 case "Wait":
-                case "Cancelling":
+                case "Canceling":
                     return (Brush)resourceDictionary["$RunningStatusColor"];
```

Comment:
is this "Canceling" will be same as what will come from ENUM value or other?

Reply:
Yes right it is coming from Enum, need to change enum value as well.